float64[] forces
geometry_msgs/Vector3[] positions
geometry_msgs/Vector3[] normals
